test/support/fixtures.js

```diff
@@ -1,6 +1,14 @@
-'use strict'
+import { join } from 'path'
+import { lists } from '../fixtures/lists'
+import { project } from '../fixtures/projects'
+import { tags } from '../fixtures/tags'
+import { selections } from '../fixtures/selections'
+import { items } from '../fixtures/items'
+import { metadata } from '../fixtures/metadata'
+import { notes } from '../fixtures/notes'
+import { ontology } from '../fixtures/ontology'
+import { photos } from '../fixtures/photos'
 
-const { join } = require('path')
 const ROOT = join(__dirname, '..', 'fixtures')
```

Comment:
`__dirname` does not work in ES modules. you can get a file URL to the module from `import.meta.url`